packages/cli-platform-apple/src/config/findPodfilePath.ts

```diff
@@ -19,14 +19,17 @@ const IOS_BASE = 'ios';
 // Podfile in the bundle package
 const BUNDLE_VENDORED_PODFILE = 'vendor/bundle/ruby';
 
-export default function findPodfilePath(cwd: string) {
+export default function findPodfilePath(
+  cwd: string,
+  platformName: string = IOS_BASE,
```

Comment:
Looks like we can get rid of IOS_BASE variable
```suggestion
  platformName: string = 'ios',
```